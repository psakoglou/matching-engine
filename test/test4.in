BUY GFD 1000 10 ORDER1
BUY GFD 999 10 ORDER2
BUY GFD 998 10 ORDER3
SELL GFD 1001 10 ORDER4
SELL GFD 1002 10 ORDER5
PRINT
CANCEL ORDER1
MODIFY ORDER2 BUY 1003 10
MODIFY ORDER3 BUY 1004 10