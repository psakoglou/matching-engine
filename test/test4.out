SELL:
1001 10
1002 10
BUY:
1000 10
999 10
998 10
TRADE ORDER2 1003 10 ORDER4 1001 10
TRADE ORDER3 1004 10 ORDER5 1002 10